runelite-client/src/main/java/net/runelite/client/plugins/agility/Obstacles.java

```diff
@@ -217,7 +217,9 @@
 		// Weiss
 		LITTLE_BOULDER, ROCKSLIDE_33184, ROCKSLIDE_33185, NULL_33327, NULL_33328,
 		// Al-Kharid
-		BROKEN_WALL_33344, BIG_WINDOW
+		BROKEN_WALL_33344, BIG_WINDOW,
+		//Taverley
+		STILE
```

Comment:
Is this the correct stile?

Reply:
Yes, it highlights the stile as shown in the image above.

Reply:
Needs rebase then

Reply:
This PR now changes 24 files and has 500 lines of diff...

Reply:
This was because I rebased.  Do you know how I can correct this?

Reply:
You've not rebased correctly then. You should try doing a hard reset to your last commit before the rebase, then doing `git fetch upstream && git rebase upstream/master`. Then you will have to resolve merge conflicts as your commits are applied upon that.